### PR TITLE
Update PyMongo and Werkzeug

### PR DESCRIPTION
**Pull request by torinkwok**

### CODE REVIEW

File: docsets/Werkzeug/docset.json [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
+        "werkzeug"
+    ],
+    "specific_versions": [
+        {
+            "version": "0.16.1",
+            "archive": "versions/0.16.1/Werkzeug.tgz"
+        },
+        {
+            "version": "0.9.6",
+            "archive": "versions/0.9.6/Werkzeug.tgz"
```
Comment (Kapeli) [inline docsets/Werkzeug/docset.json]: You need to add this archive as well.

File: docsets/PyMongo/docset.json [hunk 2]
```diff
@@ -1,15 +1,19 @@
 {
     "name": "PyMongo",
-    "version": "3.9.0",
+    "version": "3.10.1",
     "archive": "PyMongo.tgz",
```
Comment (Kapeli) [inline docsets/PyMongo/docset.json]: You need to update the archive at this path as well.

File: docsets/Werkzeug/docset.json [hunk 3]
```diff
@@ -1,10 +1,22 @@
 {
     "name": "Werkzeug",
-    "version": "0.9.6",
+    "version": "0.16.1",
     "archive": "Werkzeug.tgz",
```
Comment (Kapeli) [inline docsets/Werkzeug/docset.json]: You need to update the archive at this path as well.

Comment (torinkwok) [reply docsets/Werkzeug/docset.json]: Done. https://github.com/Kapeli/Dash-User-Contributions/pull/2675/commits/ea0a9e474f4969e522905e8b4a369bf27579922e

File: docsets/PyMongo/docset.json (hunk 2 above)
Comment (torinkwok) [reply docsets/PyMongo/docset.json]: Done. https://github.com/Kapeli/Dash-User-Contributions/pull/2675/commits/ec505531b52d5aea60fc5bd708eaaeb7770a44aa

File: docsets/Werkzeug/docset.json (hunk 1 above)
Comment (torinkwok) [reply docsets/Werkzeug/docset.json]: Done. https://github.com/Kapeli/Dash-User-Contributions/pull/2675/commits/ed7f0d33f3eb351abb6d5fe23b9513733ceef510

**Comment by Kapeli** [comment]
Thank you! 👍

**Pull request merged**